src/Lusid.Instruments.Examples/Utilities/InstrumentExamples.cs

```diff
@@ -58,7 +58,7 @@ public static LusidInstrument GetExampleInstrument(string instrumentName)
             };
         }
 
-        internal static LusidInstrument CreateExampleFxForward(bool isNdf = true)
+        internal static FxForward CreateExampleFxForward(bool isNdf = true)
```

Comment:
All the changes in this file just allow my to access more information on the instrument by knowing the specific type.